x/distribution/types/fee_pool.go

```diff
@@ -8,46 +8,18 @@ import (
 
 // global fee pool for distribution
 type FeePool struct {
-	TotalValAccum TotalAccum `json:"val_accum"`      // total valdator accum held by validators
-	ValPool       DecCoins   `json:"val_pool"`       // funds for all validators which have yet to be withdrawn
-	CommunityPool DecCoins   `json:"community_pool"` // pool for community funds yet to be spent
-}
-
-// update total validator accumulation factor
-// NOTE: Do not call this except from ValidatorDistInfo.TakeFeePoolRewards().
-func (f FeePool) UpdateTotalValAccum(height int64, totalBondedTokens sdk.Dec) FeePool {
-	f.TotalValAccum = f.TotalValAccum.UpdateForNewHeight(height, totalBondedTokens)
-	return f
-}
-
-// get the total validator accum for the fee pool without modifying the state
-func (f FeePool) GetTotalValAccum(height int64, totalBondedTokens sdk.Dec) sdk.Dec {
-	return f.TotalValAccum.GetAccum(height, totalBondedTokens)
+	CommunityPool sdk.DecCoins `json:"community_pool"` // pool for community funds yet to be spent
```

Comment:
Excess tokens go here from various calculations, correct? What happens with these funds? What are they used for?

Reply:
Excess tokens should be negligible; mostly this gets whatever the "Community Tax" allocation percentage is (2% for the Cosmos Hub at launch).